pkg/apis/planetscale/v2/vitessshard_types.go

```diff
@@ -308,24 +308,50 @@ type VitessShardStatus struct {
 	// HasMaster is a condition indicating whether the Vitess topology
 	// reflects a master for this shard.
 	HasMaster corev1.ConditionStatus `json:"hasMaster,omitempty"`
+
 	// HasInitialBackup is a condition indicating whether the initial backup
 	// has been seeded for the shard.
 	HasInitialBackup corev1.ConditionStatus `json:"hasInitialBackup,omitempty"`
+
 	// MasterAlias is the tablet alias of the master according to the global
 	// shard record. This could be empty either because there is no master,
 	// or because the shard record could not be read. Check the HasMaster
 	// condition whenever the distinction is important.
 	MasterAlias string `json:"masterAlias,omitempty"`
 
+	// BackupLocations reports information about the backups for this shard in
```

Comment:
Any reason for moving this? If anything, I'd probably leave this below Idle but move MasterAlias down so all the ConditionStatus-type fields are together.

Reply:
If you compare against the original file it's actually not moved. In Master right now BackupLocations is directly underneath MasterAlias.

Reply:
It's at the bottom on master: https://github.com/planetscale/vitess-operator/blob/master/pkg/apis/planetscale/v2/vitessshard_types.go#L326-L328

Reply:
oh, was apparently not looking at master then. My bad - I'll move it back. No reason why it got moved. Think it was the shuffle of removing conditions and then puttting them back.

Reply:
Moved back to the bottom and grouped all condition fields together https://github.com/planetscale/vitess-operator/pull/31/commits/c192f7b252505bee0b39f77b9bdb99cb235632fd.